### DISCUSSION TITLE
Clickable text help :)

### DISCUSSION THREAD

**Discussion by Spruce-Studios** [TownyAdvanced/Towny#7362] [Q&A] [answered]
Hello! Whenever players want to do an important action (such as deleting a town or leaving a town) they are shown a message in chat with two buttons - one green one to confirm or a red one to cancel.

I have a few questions:

1) Is it possible to remove these buttons so that players are required to type the command?

2) When clicking the button - what actually happens? We have a plugin that blocks commands from being run by users unless they are whitelisted commands. "/confirm" is on our whitelist, and it works when typed, but when the button is clicked it doesn't work.

Thanks in advance :)

**Comment by LlmDl** [comment, marked as answer]
If you have a plugin blocking commands that are clicked, or are blocking commands with plugin prefixes (which towny's commands have,) you want to white list the `towny:` prefix in that plugin. Whitelisting `towny:confirm` and `towny:cancel` is a good move.
